<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>购物系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="outerLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="margin">
     <number>12</number>
    </property>
    <!-- Left Sidebar -->
    <item>
     <layout class="QVBoxLayout" name="leftSidebarLayout">
      <item>
      <widget class="QPushButton" name="homeButton">
       <property name="text">
        <string>首页</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
        </item>
        <item>
      <widget class="QPushButton" name="enterMallButton">
       <property name="text">
        <string>进入商城</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
        </item>
        <item>
       <widget class="QPushButton" name="accountButton">
        <property name="text">
         <string>账号</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ordersButton">
        <property name="text">
         <string>账单</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="chatButton">
        <property name="text">
         <string>客服</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cartButton">
        <property name="text">
         <string>购物车</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="leftSidebarSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <!-- Main Content -->
    <item>
     <layout class="QVBoxLayout" name="rootLayout">
      <item>
       <layout class="QHBoxLayout" name="topBarLayout">
        <item>
         <widget class="QLineEdit" name="searchInput">
          <property name="placeholderText">
           <string>搜索商品、店铺</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>搜索</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="topBarSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="carouselArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
     </widget>
      </item>
      <item>
       <widget class="QWidget" name="recommendationsArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
     </widget>
      </item>
      <item>
       <widget class="QWidget" name="promotionsArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bottomPagerLayout">
    <item>
     <widget class="QLabel" name="pageInfoLabel">
    <property name="text">
     <string>第 1/1 页</string>
    </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totalInfoLabel">
    <property name="text">
     <string>共 0 条</string>
    </property>
     </widget>
    </item>
    <item>
     <spacer name="pagerSpacer">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="sizeHint" stdset="0">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="gotoLabel">
    <property name="text">
     <string>跳转到</string>
    </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="gotoPageSpin">
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>16777215</height>
     </size>
    </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gotoPageButton">
    <property name="text">
     <string>跳转</string>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>0</height>
     </size>
    </property>
     </widget>
    </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 </ui>